### ISSUE TITLE
Counter for tasks

### ISSUE THREAD

**Issue by NolSpencer** [cseitz/SoftwareEngineering-Team-TGMGPA#51]
We can make a counter for how many tasks you have completed and this can go along with the account feature. This way you can keep track of how much progress you are making on your stuff you have to get done. long example of this would be like "You have completed 4 out 6 tasks!" or we could just have a counter 4/6 tasks done. This counter will keep a record for all time maybe?

We need 2 things:
1. Counter on today and tomorrow for tasks marked completed out of total tasks on that side
2. Separate (not tied to any day) counter for "X tasks completed in the past week". Access this data by doing `TaskList.completed(7)`

**Comment by cseitz** [comment]
Great idea! I'll add this to the sprint backlog.

**Comment by wesley-ka** [comment]
I added all the functionality as specified here. 

One thing whoever is going to review this is going to notice is that this isn't pretty, as a matter of fact, one might notice that I am not the developer with the sharpest sense of design.

A solution that I found for this some time ago is if you give me a mockup I can pretty much get the component to look exactly like that, but if you leave to my imagination... well, you are seeing what happens.

And as always, if some extra functionality is needed or a new idea pops up, let me know and I will get to work into adding it right away.
